hooks/command

```diff
@@ -80,6 +80,11 @@ if [ $has_errors -eq 0 ]; then
     create_annotation=1
   fi
 
+  if [[ -n "${BUILDKITE_PLUGIN_JUNIT_ANNOTATE_REPORT_SLOWEST}" ]]; then
```

Comment:
I would suggest providing a default (empty) value in case the variable is not set:

```suggestion
  if [[ -n "${BUILDKITE_PLUGIN_JUNIT_ANNOTATE_REPORT_SLOWEST:-}" ]]; then
```